docs/4.0. Behaviour - Authorization Server.md

```diff
@@ -6,66 +6,84 @@ _(c) AMWA 2019, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
 
 ### Authorization Server Public Key
 
-The Authorization Server MUST provide all public keys used for signing tokens at the `/jwks` endpoint of the API. The
-Authorization Server MAY present more than one key on this endpoint, with each key being an entry in an array.
+The Authorization Server MUST provide all public keys used for signing tokens at a non-protected endpoint, the location
+of which will correspond with the value of the the "jwks_uri" property at the server metadata endpoint. The default
+value of this endpoint on the API is `/jwks`.
 
-All public keys must be presented using the text representation used by The Secure Shell (SSH) Public Key File Format
-in [RFC 4716][RFC-4716].
+The format of the keys SHOULD be the JSON Web Key (JWK) format described in Section 4 of [RFC 8414][RFC-8414] and
+SHOULD form a JSON Web Key Set (JWKS) described in Section 5 of [RFC 8414][RFC-8414]. The Authorization Server MAY
+present more than one key within a JSON Web Key Set, with each key being an entry in the "keys" array.
 
-Any public keys used to sign tokens for NMOS APIs MUST use a `kid` (Key ID) matching the following regular expression.
-The trailing integer in this expression is intended to be an integer timestamp matching the issue time of the key.
+Any JSON web Keys used to sign tokens for NMOS APIs MUST use a `kid` (Key ID) matching the following regular
+expression. The trailing integer in this expression is intended to be an integer timestamp matching the issue time of
+the key to facilitate key rotation.
 ```
 ^x-nmos-\d+$
 ```
 
 ### Changing Keys
 
 When transitioning to a new public/private key pair used for signing tokens the Authorization Server SHOULD provide
-both the old and new public key at the `/jwks` endpoint until all tokens that may be verified by the old public key
+both the old and new public key at the public key endpoint until all tokens that may be verified by the old public key
 would have expired. However, if a private key is known to be compromised, the Authorization Server MUST remove it
-from the `/jwks` endpoint immediately.
+from the public key endpoint immediately.
 
-Authorization Servers SHOULD provide new public keys on the `/jwks` endpoint for at least 2 hours before issuing tokens
-signed by the corresponding private key to allow time for clients to cache the new public key.
+Authorization Servers SHOULD provide new public keys at the public key endpoint endpoint for at least 2 hours before
```

Comment:
Note to self, make sure that the client key refresh requirements match this time period